WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -0,0 +1,166 @@
+import UIKit
+
+class SupportTableViewController: UITableViewController {
+
+    // MARK: - Properties
+
+    var sourceTag: SupportSourceTag?
+    private var tableHandler: ImmuTableViewHandler!
+
+    // MARK: - Init
+
+    override init(style: UITableViewStyle) {
```

Comment:
Hey @jklausa . `override init(style: UITableViewStyle)` is needed because `required convenience init()` sets the table to grouped. It errors without it. I don't see a way around that. If you do, please let me know.